### ISSUE TITLE
Release an i386 (x86) compatible version of Enclave

### ISSUE THREAD

**Issue by enclave-marc-barry** [enclave-networks/install.enclave.io#11]
At the moment we don't build `i386` compatible (x86) versions of Enclave for Linux or Windows. Our build server needs to produce the appropriate packages and on Windows we need to ensure distribution an x86 version of the TAP driver.

At the time of writing we're only building for Linux targeting `arm`, `arm64` and `x64` and Windows targeting `x64`.

Related issue: [Unable to find expected entry 'main/binary-i386/Packages' in Release file](https://github.com/enclave-networks/install.enclave.io/issues/10)

This issue is **on hold** until a paying customer requests `i386` builds, or it will unblock a sale.

**Comment by enclave-alistair** [comment]
Tagging this as "wontfix", to reflect the fact we won't do this unless someone pays us for it, directly or not.

.NET does not support x86 Linux, and will not do so in the near term. Add that to the fact that modern Windows OSes do not support x86, and the use of x86 becomes a bit of an edge case.

**Comment by enclave-marc-barry** [comment]
Agreed. Leaving a note for future self to remind me that we'll advise customers to provide access to legacy devices via Exit Node feature

**Comment by enclave-alistair** [comment]
Closing this; we've added support for x86 windows while we were adding arm64 support. Linux will never have x86 support in .NET, so that point is a bit moot.
